tabbycat/notifications/views.py

```diff
@@ -125,14 +125,16 @@ def post(self, request, *args, **kwargs):
 
         data = json.loads(request.body)
 
-        records = SentMessage.objects.filter(hook_id__in=[obj['hook-id'] for obj in data if 'hook-id' in obj])
+        # Ignore all objects without a Tabbycat-specified hook ID
+        data = [obj for obj in data if 'hook-id' in obj and obj['hook-id'] is not None]
```

Comment:
The hook-id is never None, as it is there only when specified. `and obj['hook-id'] is not None` is not needed.

Reply:
I thought about it, and I realized we don't actually know this. It's coming from an external system, so you can't make any assumptions about the input (_i.e._, you should always check). It's technically an open URL, so I could even write a short script on my computer to POST to the webhook address with `{'hook-id': null}`, and then you'd have a `obj['hook-id']` that's `None`.

In this case it's not really a security risk, per se, because the consequence is pretty minor and the request can't modify important data anyway. But this is the kind of thing that I think about when interfacing with external systems.